packages/shared-business/src/constants/countries.ts

```diff
@@ -1,7 +1,10 @@
+import { Lazy } from "@swan-io/boxed";
 import { isNotNullish } from "@swan-io/lake/src/utils/nullish";
 import { CountryItem } from "../components/CountryPicker";
 import { t } from "../utils/i18n";
 
+type LazyValue<T> = ReturnType<typeof Lazy<T>>;
```

Comment:
makes me think we might want to add a `Lazy<T>` type in Boxed for that

Reply:
yes will be nice

Reply:
thinking. what's the metric for

> file import is reduced by 30%

?

Reply:
hum, I'd say that's pretty negligible to change. I feel like we have room for improvement in other areas that would have way more impact (e.g. alternative way to load translations so that we don't load all locales)

Reply:
yes this is why I prefixed by suggestion. I've got this idea by discovering `Lazy` but wasn't sure you'll find it really usefull.  
By the way I can start thinking about a better way to load translations in another PR 👍